meta.json

```diff
@@ -1,9 +1,9 @@
 {
```

Comment:
Note that this file changed names from `meta_cuda.json` to just `meta.json` and was moved to the default location.